content/general/concepts/big-o-notation/big-o-notation.md

```diff
@@ -46,4 +46,4 @@ def foo(list1, list2):
 foo(["Hello", "World"], ["Code", "Ninjas"])
```

Comment:
I believe Code Ninjas is some company that does coding boot camps for kids: https://www.codeninjas.com/

Can we use another example?